src/patternfly/components/DataToolbar/examples/data-toolbar-selected-filters-example.hbs

```diff
@@ -0,0 +1,75 @@
+
+    {{!-- Toggle group --}}
+    {{#> data-toolbar-group data-toolbar-group--modifier="pf-m-toggle-group pf-m-reveal-on-xl" data-toolbar-group--IsExpanded="true"}}
+      {{> data-toolbar-toggle data-toolbar-toggle--SelectedFilters="6"}}
+
+      {{!-- Search filter --}}
+      {{> data-toolbar-item-search-filter button--id="content"}}
+
+      {{!-- Filter group --}}
+      {{#> data-toolbar-group data-toolbar-group--modifier="pf-m-filter-group"}}
+        {{#> data-toolbar-item}}
+          {{#> select id=(concat data-toolbar--id '-select-checkbox-status') select--IsCheckboxSelect="true" select--ItemIsSelected="true"}}
+            Status
+          {{/select}}
+        {{/data-toolbar-item}}
+        {{#> data-toolbar-item}}
+          {{#> select id=(concat data-toolbar--id '-select-checkbox-risk') select--IsCheckboxSelect="true" select--ItemIsSelected="true"}}
+            Risk
+          {{/select}}
+        {{/data-toolbar-item}}
+      {{/data-toolbar-group}}
+    {{/data-toolbar-group}}
+
+    {{#> data-toolbar-item}}
+      {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Sort"'}}
+        <i class="fas fa-sort-amount-down" aria-hidden="true"></i>
+      {{/button}}
+    {{/data-toolbar-item}}
+
+    {{!-- Overflow menu --}}
+    {{#> data-toolbar-item data-toolbar-item--modifier="pf-m-overflow-menu"}}
+      {{!-- Uncomment when overflow menu merges --}}
+      {{!-- {{> overflow-menu-action-group-example overflow-menu--IsSingle="true" overflow-menu--RevealOn="xl" overflow-menu-control--HasAdditionalOptions="true" overflow-menu--id=data-toolbar--id}} --}}
+
+
+      {{!-- Remove when overflow menu merges --}}
+      Overflow menu
+    {{/data-toolbar-item}}
+  {{/data-toolbar-content}}
+
+  {{!-- Expanded content --}}
+  {{#> data-toolbar-expandable-content data-toolbar-expandable-content--modifier="pf-m-reveal-on-xl" data-toolbar-expandable-content--IsExpanded="true"}}
+
+    {{!-- Search filter --}}
+    {{> data-toolbar-item-search-filter button--id="expanded-content" data-toolbar-item-search-filter--modifier="pf-m-shared-item"}}
+
+    {{!-- Filter --}}
+    {{#> data-toolbar-item data-toolbar-item--modifier="pf-m-shared-item"}}
+      {{#> select id=(concat data-toolbar--id '-select-checkbox-status-expanded')}}
+        Status
+      {{/select}}
+    {{/data-toolbar-item}}
+
+    {{> data-toolbar-item-chip-group chip-group--label="Status" chip-group--id=(concat data-toolbar--id '-group1-')}}
+
+    {{!-- Filter --}}
+    {{#> data-toolbar-item data-toolbar-item--modifier="pf-m-shared-item"}}
+      {{#> select id=(concat data-toolbar--id '-select-checkbox-risk-expanded') select--IsCheckboxSelect="true"}}
+        Risk
+      {{/select}}
+    {{/data-toolbar-item}}
+
+    {{> data-toolbar-item-chip-group chip-group--label="Risk" chip-group--id=(concat data-toolbar--id '-group2-')}}
+
+    {{!-- Clear all filters button --}}
+    {{#> data-toolbar-item data-toolbar-item--modifier="pf-m-clear"}}
```

Comment:
Consulted @mceledonia, he says align it right. The button looks misaligned when aligned left. @kybaker @mcarrano wyt?

Reply:
ya I saw that to b/c of the spacing around the button. I wonder if this is showing a larger issue of the type of button. I'd guess it will come up again, it feels like a problem with the button itself if we are trying to hide it by aligning it left, but I understand that for the target area we might be stuck with it.